Util/Docker/Carla.Dockerfile-mod

```diff
@@ -0,0 +1,39 @@
+FROM carla-prerequisites:latest as build-env
```

Comment:
Can I find a description of the purpose of the file anywhere?

Reply:
The Prerequisites.Dockerfile installs the build tools and pre-builds the Unreal Engine such that the Carla.Dockerfile only needs to build Carla and nothing else. It's like a build environment setup.